src/v1/sources/shopify/pixelTransform.js

```diff
@@ -77,6 +77,8 @@ function processPixelEvent(inputEvent) {
     version: '2.0.0',
   });
   message.setProperty('context.topic', name);
+  message.setProperty('context.shopifyDetails', data);
```

Comment:
why not entire raw payload to be maintained in `context.shopifyDetails` instead of only `data` object ?

Reply:
adding the entire raw payload sent over by Shopify here